src/main/java/com/blibli/oss/qa/util/services/NetworkListener.java

```diff
@@ -53,6 +52,18 @@ public NetworkListener(WebDriver driver, String harFileName) {
         }
         createHarBrowser();
     }
+    public NetworkListener(WebDriver driver , DevTools devTools , String harFileName){
+        this.devTools = devTools;
+        this.driver = driver;
+        this.harFile = harFileName;
+        try {
+            Files.delete(java.nio.file.Paths.get(harFile));
+        } catch (IOException e) {
+            //ignored
```

Comment:
i think its better add log might be issue on future(?)

Reply:
added print stack trace for debugging purpose if there are some error